lookout/style/format/analyzer.py

```diff
@@ -145,58 +138,15 @@ def group_line_nodes(y, y_pred, vnodes_y, rule_winners):
                         find_deleted_lines(prev_file, file),
                     )))
                 fe = FeatureExtractor(language=lang, **rules.origin_config["feature_extractor"])
-                res = fe.extract_features([file], [lines])
-                if res is None:
+                feature_extractor_output = fe.extract_features([file], [lines])
+                if feature_extractor_output is None:
                     if self.config["report_parse_failures"]:
-                        comment = Comment()
-                        comment.file = file.path
-                        comment.confidence = 100
-                        comment.line = 1
-                        comment.text = "Failed to parse this file"
-                        comment.append(comment)
+                        comments.append(
+                            generate_comment(file.path, 100, 1, "Failed to parse this file"))
```

Comment:
This comment should be put on line #0 - this is a special case which is properly handled by Lookout